src/Workleap.DomainEventPropagation.Abstractions/Workleap.DomainEventPropagation.Abstractions.csproj

```diff
@@ -11,6 +11,9 @@
     <AssemblyOriginatorKeyFile>../Workleap.DomainEventPropagation.snk</AssemblyOriginatorKeyFile>
     <RootNamespace>Workleap.DomainEventPropagation</RootNamespace>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
```

Comment:
From my understanding this is to enable warnings from C# 10, wouldn't we want to have this all the time given we indicate --WarnAsError when we build in the CI?

Reply:
Did you mean to use AnalysisLevel like in Publishing?